test/tst_input.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+// A class to hold test data, an event type/key/modifiers
+// as used in QKeyEvent and a matching Neovim input string.
+class InputTest
```

Comment:
`struct InputTest`?

We're using this like a struct, and a class with all public members is basically a struct anyways.

Reply:
yup, updating.